src/index.js

```diff
@@ -1,6 +1,9 @@
 const fs = require('fs');
 const path = require('path');
 const rimraf = require('rimraf');
+const debug = require('debug');
+
+const d = debug('electron-forge:packager:languages');
```

Comment:
@thomasbachem, `electron-packager-languages` is a standalone package that's not directly associated with "electron-forge" or something similar. Therefore, I think it's better to have a debug namespace of `electron-packager-languages`.